webpack.config.js

```diff
@@ -3,7 +3,6 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
```

Comment:
For better debug you can use `devtool: 'source-map'` (allow set breaking points on `ts` files from chrome dev tools)